File: jungu/1924.cpp
#include <iostream>
#include <cstdio>
#include <string>

using namespace std;
int day[12] = {31,28,31,30,31,30,31,31,30,31,30,31};
string days[7] = { "SUN","MON","TUE","WED","THU","FRI","SAT" };

int main()
{
	int m, d;
	int sum = 0;
	cin >> m >> d;
	for (int i = 0; i<m - 1; i++)
	{
		sum += day[i];
	}
	sum += d;
	printf("%s\n", days[sum % 7].c_str());

}

File: yoh/lec01/1924.py
DAYS = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT']
x_days = 0
x, y = map(int, raw_input().split())
for i in range(1, x):
	if i == 2:
		x_days += 28
	elif i in [4, 6, 9, 11]:
		x_days += 30
	else:
		x_days += 31
print DAYS[(x_days + y) % 7]